test/suite/test_oligarch15.py

```diff
@@ -56,6 +56,21 @@ def conn_extensions(self, extlist):
     def early_setup(self):
         os.mkdir('kv_home')
 
+    # Restart the node without local files
+    def restart_without_local_files(self):
+        # Close the current connection
+        self.close_conn()
+
+        # Remove local files
+        for f in os.listdir():
```

Comment:
Can we rename instead of delete to ease debugging?

Reply:
Good idea! Done in [3dfd18f](https://github.com/wiredtiger/wiredtiger/pull/11250/commits/3dfd18f463cde9f0a44a06a80622d307400eb262).